time.go

```diff
@@ -0,0 +1,24 @@
+package testkit
+
+import (
+	"fmt"
+	"time"
+)
+
+// TimeAdvancer is a function that determines the new time that the engine is
+// advanced to during a call to Test.AdvanceTime().
+type TimeAdvancer func(before time.Time) (after time.Time, description string)
+
+// ToTime returns a TimeAdvance that advances the engine time to a specific time.
```

Comment:
Am I correct in understanding `a TimeAdvance` as `an instance of the TimeAdvance function`?

Reply:
Nah, that should read:

```suggestion
// ToTime returns a TimeAdvancer that advances the engine time to a specific time.
```

Reply:
Well, so I guess, yes you are correct, but also I misspelled it :)